benchmark/run_performance_test.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+# Usage ./run_performance_test <path/to/coniql/venv> <number-of-clients>
+#   <path/to/coniql/venv> = path to the location of the python virtual 
+#                           environment where Coniql has been installed
+#   <number-of-clients>   = number of websocket clients to start up
+
+
+# Get command line arguments
+args=("$@")
+
+# Some parameters
+SUB_DIR="benchmark"
+N_SAMPLES=1000
+# 1 = old, 2 = new
+PROTOCOL=2
```

Comment:
Is it worth turning these into more parameters for the script? Its nicer to have everything you may want to change as a command line parameter rather than needing to edit the script itself.

Reply:
I've improved the parsing of command line arguments in the script and also added these as parameters for the script.